plugin/web/styles/components/Layout/_wdContent.less

```diff
@@ -0,0 +1,9 @@
+.ant-layout-content{    
```

Comment:
If this makes the most sense to you feel free to keep it as you have it now, but if you are recreating the same directory structure here for styles as you are for the components, would it not be easier for you to just keep the XX.less files in the same folder as the component that uses it?
If you do as you are doing now, it might be hard to keep the directory structures in-sync